en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -0,0 +1,404 @@
+    [output_adapter.email]
+    from_address= "wso2iamtest@gmail.com"
+    username= "wso2iamtest"
+    password= "Wso2@iam70"
+    hostname= "smtp.gmail.com"
+    port= 587
+    enable_start_tls= true
+    enable_authentication= true
+    ```    
+    
+2. Add the following properties to the `deployemnt.toml` to configure notification handling and 
+self registration to support multiple notification channels.
+
+    ```toml
+    [identity_mgt.notification]
+    default_notification_channel = "EMAIL"
+    enable_resolve_notification_channel = true
+    
+    [identity_mgt.user_self_registration]
+    SkipAccountLockOnVerifiedPreferredChannel = false
+    enable_detailed_api_response = true
+    ```    
+    
+    <table>
+    <tr>
+        <th>Property Name</th>
+        <th>Functionality</th>
+    </tr>
+    <tr>
+    <td>default_notification_channel</td>
+    <td>
+        <ul>
+            <li>Default notification channel. If the user has not specified a notification channel
+            user will be notified with the defined channel.</li>
+            <div class="admonition warning">
+                <p class="admonition-title">warning</p>
+                <p>
+                    Currently, WSO2 IS supports **SMS** and **EMAIL** as Internal Notification channels.
+                    **DO NOT** change the values since they are case sensitive. Changing the case will
+                    lead for Errors. 
+                </p>
+            </div>
+            <li>By default this value is set to <code>EMAIL</code>.</li>
+        </ul>
+    </td>
+    </tr>
+    <tr>
+    <td>enable_resolve_notification_channel</td>
+    <td>
+        <ul>
+        <li>Enable server to determine whether to send notifications via a SMS or an email.</li> 
+        <li>Disabling this property would send  notifications via the channel defined by 
+        <code>default_notification_channel</code>.</li>
+        </ul> 
+    </td>
+    </tr>
+    <tr>
+    <td>skip_account_lock_on_verified_preferred_channel</td>
+    <td>
+        <ul>
+            <li>Allows a user to skip account confirmation if the user is already veirifed.</li>
+            <div class="admonition note">
+                <p class="admonition-title">Note</p>
+                <p>
+                    WSO2 Recommends to verify users after self registration than 
+                    pre verified user self registration. Therefore, it is recommended to set the property to 
+                    <code>false</code>. 
+                </p>
+            </div>
+        </ul>
+    </td>
+    </tr>
+    <tr>
+    <td>enable_detailed_api_response</td>
+    <td>
+        <ul>
+            <li>Enable detailed API response body.</li> 
+            <li>By default this value is set to <code>false</code>.</li>
+        <ul>
+    </td>
+    </tr>
+    </table>
+    
+    !!! tip
+            The **AccountConfirmation** and **SMSAccountConfirmation** templates are 
+            used to send this email notifications.
```

Comment:
this --> these

Reply:
Fixed: 030f591d9fd2591616bce502af05234321811549